src/redux/books/books.js

```diff
@@ -0,0 +1,23 @@
+import { createReducer, createAction } from '@reduxjs/toolkit';
+
+// Define action type for addding & removing
+export const ADD_BOOK = createAction('books/add');
+export const REMOVE_BOOK = createAction('books/remove');
+
+// set intial state to an empty array of books
+const initialState = {
+  books: [],
+};
+
+// Reducer
+const booksReducer = createReducer(initialState, (builder) => {
+  builder.addCase(ADD_BOOK, (state, action) => {
+    state.books.push(action.payload);
+  });
+  builder.addCase(REMOVE_BOOK, (state, action) => {
+    state.books.filter((books) => books.id !== action.payload);
```

Comment:
- [ ] I see you are not returning anything, it is necessary for you to return so that the function can be broken out from where the case is true.